packages/tools/src/converter.ts

```diff
@@ -0,0 +1,7 @@
+export const convertFromDaysToMilliseconds = (days: number) => {
+	if (typeof days !== 'number') {
```

Comment:
Should we round it to the closest integer? What if someone provides a decimal here?

Reply:
3.1 days still has some equivalence on millis innit? :kek:

Good question, i can take a look, maybe putting an `IsInteger` would suffice, wdyt?